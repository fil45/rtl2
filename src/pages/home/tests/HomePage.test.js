import React from "react";
import { render } from "@testing-library/react";
import HomePage from "../HomePage";

describe("HomePage --->", () => {
  it("should render the page", () => {
    const { getByText } = render(<HomePage />);
    const welcomeElement = getByText(/WELCOME TO REACT TESTING LIBRARY DEMO/i);
    expect(welcomeElement).toBeInTheDocument();
  });


  it("should match snapshot", () => {
    const { asFragment } = render(<HomePage />);
    expect(asFragment(<HomePage />)).toMatchSnapshot();
  });
});
