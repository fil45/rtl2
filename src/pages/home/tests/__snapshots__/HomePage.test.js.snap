// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage ---> should match snapshot 1`] = `
<DocumentFragment>
  <div>
    <h2>
      WELCOME TO REACT TESTING LIBRARY DEMO!
    </h2>
    <iframe
      src="https://testing-library.com/docs/react-testing-library/intro/"
      title="RTL"
    />
  </div>
</DocumentFragment>
`;
